deploy.py

```diff
@@ -0,0 +1,251 @@
+import boto3
+import json
+import yaml
+import argparse
+import os
+
+
+def find_instances(region, clusters):
```

Comment:
```suggestion
def find_instances(region: str, clusters: List[str]):
```
type hint in the arguments, then we can drop it in the comment.